pkg/ssh/config.go

```diff
@@ -139,13 +139,16 @@ func writeSSHConfig(path, content string, log log.Logger) error {
 	return nil
 }
 
-func getSSHConfig() (string, error) {
-	homeDir, err := homedir.Dir()
-	if err != nil {
-		return "", errors.Wrap(err, "get home dir")
+func GetDefaultSSHConfigPath() (string, error) {
+	if config.ContextOptionSSHConfigPath == "" {
+		homeDir, err := homedir.Dir()
+		if err != nil {
+			return "", errors.Wrap(err, "get home dir")
+		}
+		return filepath.Join(homeDir, ".ssh", "config"), nil
+	} else {
+		return config.ContextOptionSSHConfigPath, nil
```

Comment:
If I'm not mistaken `homedir.Dir()` returns an absolutes path. Let's do the same for the `SSHConfigPath` in case it's relative